packages/next-codemod/transforms/__testfixtures__/next-async-request-api-dynamic-apis/async-prop-cast-02.output.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { 
+  headers,
+  cookies, 
+  draftMode, 
+} from 'next/headers'
+
+export function MyDraftComponent() {
+if (/* TODO: await this async call and propagate the async correctly */
```

Comment:
Where are docs for what "correctly" means here? Because it isn't just awaiting the calls. This here is likely a component. Since it uses `cookies` it must only be used as a Server Component so there's nothing left to do.

Reply:
But you don't await components.